docs/src/pages/customization/themes/themes.md

```diff
@@ -345,6 +345,12 @@ html {
 
 {{"demo": "pages/customization/themes/FontSizeTheme.js"}}
 
+### Responsive fonts
+
+You can use the `responsiveTypography` helper in `@material-ui/core/utils` to make Typography font sizes in the theme responsive.
```

Comment:
I realize many of our demos are poor in this regard, but could you expand on this a bit; i.e. explain what it means to make a font responsive, and describe what the demo shows?

Reply:
Hi @mbrookes I expanded on it now, just reopen if not enough? Trying to keep it very light still, as docs seem to be getting heavy, as @oliviertassinari also [mentioned](https://github.com/mui-org/material-ui/pull/14573#discussion_r274408962).